source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,534 @@ Filters Builders
+
+The following example matches documents that have a ``color`` field
+starting with the letter "p" in the ``paint_purchases`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin regexComparison
+   :end-before: end regexComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+   :copyable: false
+   
+   { "$regex" : "^p" }
+
+The following shows the output of the query using the operator specified above:
+   
+.. code-block:: json
+   :copyable: false
+      
+   { "_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"] }
+   { "_id": 6, "color": "pink", "qty": 5, "vendor": ["C"] }
+
+.. _bitwise:
+
+Bitwise
+-------
+
+The bitwise operators convert a number into its binary value to
+evaluate its bits.
+
+The bitwise operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Bitwise Method
+     - Matches
+      
+   * - :java-core-api:`bitsAllSet() <com/mongodb/client/model/Filters.html#bitsAllSet(java.lang.String,long)>`
+     - documents when the specified bits of a field are set (i.e. "1").
+
+   * - :java-core-api:`bitsAllClear() <com/mongodb/client/model/Filters.html#bitsAllClear(java.lang.String,long)>`
+     - documents when the specified bits of a field are clear (i.e. "0").
+      
+   * - :java-core-api:`bitsAnySet() <com/mongodb/client/model/Filters.html#bitsAnySet(java.lang.String,long)>`
+     - documents when at least one of the specified bits of a field are set (i.e. "1").
+     
+   * - :java-core-api:`bitsAnyClear() <com/mongodb/client/model/Filters.html#bitsAnyClear(java.lang.String,long)>`
+     - documents when at least one of the specified bits of a field are clear (i.e. "0").
+
+The following example matches documents that have a ``bitField`` field
+with bits set at positions of the corresponding bitmask "34" (i.e.
+"00100010") in the ``binary_numbers`` collection: 
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin bitsComparison
+   :end-before: end bitsComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+      :copyable: false
+   
+   {  "a": { "$bitsAllSet": [1,5] }}
+
+The following shows the output of the query using the operator specified above:
+   
+.. code-block:: json
+   :copyable: false
+      
+   { "_id": 9, "a": 54, "binaryValue": "00110110" }
+   { "_id": 12, "a": 102, "binaryValue": "01100110" }
+
+.. _geospatial:
+
+Geospatial
+----------
+
+The geospatial operators evaluate a specified coordinate and its
+relation to a shape or location. 
+
+The geospatial operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Geospatial Method 
+     - Matches
+      
+   * - :java-core-api:`geoWithin() <com/mongodb/client/model/Filters.html#geoWithin(java.lang.String,org.bson.conversions.Bson)>`
```

Comment:
Even fully expanded, the first column is wrapping to two lines, which looks funny for method names.  Can that be fixed?